stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -66,6 +66,19 @@ public static void main(String[] args) {
     mainInternal(args, new FuseIOBench());
   }
 
+  @Override
+  public String getBenchDescription() {
+    return "This stress bench is designed for testing the reading throughput of Fuse-based "
+        + "POSIX API.\nTo write data, run `bin/alluxio runClass alluxio.stress.cli.fuse."
+        + "fuseIOBench --operation Write`\nTo read data, run `bin/alluxio runClass "
+        + "alluxio.stress.cli.fuse.fuseIOBench --operation Read`\nYou can further adjust "
+        + "the parameters specified above. Make sure the local path is successfully mounted\n"
```

Comment:
The local path can be a local filesystem path or a mounted fuse path.

Reply:
I'm not sure why we don't add these to the doc page and just refer to that.

Reply:
we are waiting for the stressbench related docs so that shawn can add his test to it